src/CLI.java

```diff
@@ -0,0 +1,88 @@
+import java.io.*;
+
+public class CLI implements UserInterface {
+    public final String GAME_GREETING = "Welcome to the Rock Paper Scissors Game.\n\n";
+    public final String CONSOLE_MOVE_REQUEST = "Please enter Rock(1), Paper(2) or Scissors(3): \n";
+    public final String CONSOLE_MOVE_DISPLAY = "You have selected: %s \n";
+    public final String AI_MOVE_DISPLAY = "AI Player selected: %s \n";
+    public final String RESULT_DISPLAY = "And the winner is: %s \n";
+    public final String REPLAY_REQUEST = "Do you want to play again? Yes(1) or No(2): \n";
+    private BufferedReader readStream;
+    private PrintStream writeStream;
+    private Game game;
+    private Symbol consoleSymbol;
+
+    public CLI(InputStream inputStream, PrintStream outputStream, Game game) {
+        this.readStream = new BufferedReader(new InputStreamReader(inputStream));
+        this.writeStream = outputStream;
+        this.game = game;
+        Symbol.setOrdinalToSymbol();
+    }
+
+    public String displayGreeting() {
+        String greeting = GAME_GREETING;
+        writeStream.println(greeting);
+        return greeting;
+    }
+
+    public Symbol requestConsoleMove() {
+        while (consoleSymbol == null) {
+            writeStream.println(CONSOLE_MOVE_REQUEST);
+            consoleSymbol = Symbol.getSymbolFromOrdinal(readInput());
+        }
+        return consoleSymbol;
```

Comment:
Shouldn't consoleSymbol just be a local? I wouldn't expect this to be a field as we now have state to worry about.